Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryOrchImplTest.java

```diff
@@ -84,28 +88,25 @@
     @Test
     public void testGetDocumentService() {
         AdapterComponentDocRepositoryOrchImpl docRepo = new AdapterComponentDocRepositoryOrchImpl();
-        assertTrue(docRepo.getDocumentService() instanceof DocumentService);
+        assertTrue(docRepo.getDocumentService() != null);
     }
 
     @Test
     public void testGetLargeFilesUtil() {
         AdapterComponentDocRepositoryOrchImpl docRepo = new AdapterComponentDocRepositoryOrchImpl();
-        assertTrue(docRepo.getLargeFileUtils() instanceof LargeFileUtils);
+        assertTrue(docRepo.getLargeFileUtils() != null);
     }
 
     @Test
     public void testGetDateUtil() {
         AdapterComponentDocRepositoryOrchImpl docRepo = new AdapterComponentDocRepositoryOrchImpl();
-        assertTrue(docRepo.getDateUtil() instanceof UTCDateUtil);
+        assertTrue(docRepo.getDateUtil() != null);
     }
 
     @Test
     public void testSetDocumentResponse_Failure() {
         AdapterComponentDocRepositoryOrchImpl docRepo = new AdapterComponentDocRepositoryOrchImpl();
-        Document doc = mock(Document.class);
-
-        when(doc.getRawData()).thenReturn(null);
-
+        DocumentMetadata doc = new DocumentMetadata();
```

Comment:
Seem  it always true here.

Reply:
This test asserts that an empty document will return false. Even though the metadata is not null, the document response should only be set if it has data associated with it.